src/utils/validators.js

```diff
@@ -8,46 +8,67 @@ const {
-*/
-static createValidationErrors(fieldDataType, errorMessage) {
-   return {
-    [`${fieldDataType}.base`]: errorMessage,
-    [`${fieldDataType}.empty`]: errorMessage,
-  };
-}
+  /**
+  * @param {object} fieldDataType data type for the field
+  * @param {object} errorMessage error message to display
+  * @returns {object} an object of validation error messages
+  */
+  static createValidationErrors(fieldDataType, errorMessage) {
+    return {
+      [`${fieldDataType}.base`]: errorMessage,
+      [`${fieldDataType}.empty`]: errorMessage,
+      'any.required': errorMessage,
+      [`${fieldDataType}.less`]: errorMessage,
+      [`${fieldDataType}.greater`]: errorMessage,
+    };
+  }
 
-/**
-* @param {object} tripRequestData one way trip request data
-* @returns {Promise<any>} a Promise of validation output
-*/
-static async validateOneWayTripRequest(tripRequestData) {
-  const whiteSpaces = /\s+/g;
-  const { createValidationErrors } = Validators;
-  const cleanString = Joi.string().replace(whiteSpaces, ' ').trim().required();
-  const schema = Joi.object({
-      travelFrom: cleanString
-      .messages(createValidationErrors('string', invalidTravelFrom)),
-      travelTo: cleanString
-      .messages(createValidationErrors('string', invalidTravelTo)),
-      travelReason: cleanString
-      .messages(createValidationErrors('string', invalidTravelReason)),
-      travelDate: Joi.date().required()
-      .messages(createValidationErrors('date', invalidTravelDate)),
-      accommodation: Joi.bool().required()
-      .messages(createValidationErrors('boolean', invalidTravelAccomodation)),
-      travelType: cleanString.lowercase()
-      .messages(createValidationErrors('string', invalidTravelType)),
-  });
-  return schema.validateAsync(tripRequestData, { abortEarly: false });
+  /**
+  * @param {object} tripRequestData one way trip request data
+  * @returns {Promise<any>} a Promise of validation output
+  */
+  static async validateOneWayTripRequest(tripRequestData) {
+    const whiteSpaces = /\s+/g;
+    const { createValidationErrors } = Validators;
+    const cleanString = Joi.string().replace(whiteSpaces, ' ').trim().required();
+    const schema = Joi.object({
+        travelFrom: cleanString
+        .messages(createValidationErrors('string', invalidTravelFrom)),
+        travelTo: cleanString
+        .messages(createValidationErrors('string', invalidTravelTo)),
+        travelReason: cleanString
+        .messages(createValidationErrors('string', invalidTravelReason)),
+        travelDate: Joi.date().required()
+        .messages(createValidationErrors('date', invalidTravelDate)),
+        accommodation: Joi.bool().required()
+        .messages(createValidationErrors('boolean', invalidTravelAccomodation)),
+        travelType: cleanString.lowercase()
+        .messages(createValidationErrors('string', invalidTravelType)),
+    });
+    return schema.validateAsync(tripRequestData, { abortEarly: false });
```

Comment:
Why did you have to work on these validations in your task?

Reply:
@descholar-ceo  I didn't, I just inserted tabs before each line and that's why it looks like I added something

Reply:
It is just formatting the code to make look better

Reply:
tha is cool